### ISSUE TITLE
useless-assert: negative-nature assertions for unsigned values

### ISSUE THREAD

**Issue by Antonboom** [Antonboom/testifylint#133]
From https://github.com/Antonboom/testifylint/pull/126:

```go
	// These one will be reported by incorrect-assert.
	ignoredAssertions = append(ignoredAssertions,
		Assertion{Fn: "Negative", Argsf: "uint(a)"},
		Assertion{Fn: "Less", Argsf: "uint(a), 0"},
		Assertion{Fn: "True", Argsf: "uint(a) < 0"},
		Assertion{Fn: "True", Argsf: "0 > uint(a)"},
		Assertion{Fn: "False", Argsf: "uint(a) >= 0"},
		Assertion{Fn: "False", Argsf: "0 <= uint(a)"},
	)
```

**Comment by Antonboom** [comment]
```
--- a/analyzer/testdata/src/debug/useless_assert_test.go
+++ b/analyzer/testdata/src/debug/useless_assert_test.go
@@ -1,6 +1,7 @@
 package debug
 
 import (
+       "fmt"
        "testing"
 
        "github.com/stretchr/testify/assert"
@@ -23,3 +24,16 @@ func TestUselessAsserts(t *testing.T) {
        assert.Zero(t, "")
        assert.Zero(t, nil)
 }
+
+func TestUselessUnsigned(t *testing.T) {
+       for _, a := range []int{-1, 0, 1} {
+               t.Run(fmt.Sprintf("%d", a), func(t *testing.T) {
+                       assert.Negative(t, uint(a))
+                       assert.Less(t, uint(a), uint(0))
+                       assert.True(t, uint(a) < 0)
+                       assert.True(t, 0 > uint(a))
+                       assert.False(t, uint(a) >= 0)
+                       assert.False(t, 0 <= uint(a))
+               })
+       }
+}
~
```

**Comment by Antonboom** [comment]
UPD: do not support `True/False` in favor of `compare` checker. Support only `Less/Greater/...`
